notebooks/community/prediction/feature_store_integration/prediction_feature_store_integration.ipynb

```diff
@@ -0,0 +1,1485 @@
+{
```

Comment:
<p>Split this into individual cells, in the order:</p><p> </p><p>Objective</p><p>Dataset</p><p>Model</p><p>Costs -- missing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1568/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done.</p>